ext/pgsql/pgsql.stub.php

```diff
@@ -112,122 +112,122 @@ function pg_affected_rows($result): int {}
+function pg_field_table($result, int $field, bool $oid_only = false): string|int|false {}
 
 /** @param resource $result */
-function pg_field_name($result, int $field_number): string {}
+function pg_field_name($result, int $field): string {}
 
 /**
  * @param resource $result
  * @alias pg_field_name
  * @deprecated
  */
-function pg_fieldname($result, int $field_number): string {}
+function pg_fieldname($result, int $field): string {}
 
 /** @param resource $result */
-function pg_field_size($result, int $field_number): int {}
+function pg_field_size($result, int $field): int {}
 
 /**
  * @param resource $result
  * @alias pg_field_size
  * @deprecated
  */
-function pg_fieldsize($result, int $field_number): int {}
+function pg_fieldsize($result, int $field): int {}
 
 /** @param resource $result */
-function pg_field_type($result, int $field_number): string {}
+function pg_field_type($result, int $field): string {}
 
 /**
  * @param resource $result
  * @alias pg_field_type
  * @deprecated
  */
-function pg_fieldtype($result, int $field_number): string {}
+function pg_fieldtype($result, int $field): string {}
 
 /** @param resource $result */
-function pg_field_type_oid($result, int $field_number): string|int {}
+function pg_field_type_oid($result, int $field): string|int {}
 
 /** @param resource $result */
-function pg_field_num($result, string $field_name): int {}
+function pg_field_num($result, string $field): int {}
 
 /**
  * @param resource $result
  * @alias pg_field_num
  * @deprecated
  */
-function pg_fieldnum($result, string $field_name): int {}
+function pg_fieldnum($result, string $field): int {}
 
 /**
  * @param resource $result
- * @param string|int $row_number
+ * @param string|int $row
  */
-function pg_fetch_result($result, $row_number, string|int $field = UNKNOWN): string|false|null {}
+function pg_fetch_result($result, $row, string|int $field = UNKNOWN): string|false|null {}
 
 /**
  * @param resource $result
- * @param string|int $row_number
+ * @param string|int $row
  * @alias pg_fetch_result
  * @deprecated
  */
-function pg_result($result, $row_number, string|int $field = UNKNOWN): string|false|null {}
+function pg_result($result, $row, string|int $field = UNKNOWN): string|false|null {}
 
 /**
  * @param resource $result
  */
-function pg_fetch_row($result, ?int $row_number = null, int $result_type = PGSQL_NUM): array|false {}
+function pg_fetch_row($result, ?int $row = null, int $mode = PGSQL_NUM): array|false {}
 
 /**
  * @param resource $result
  */
-function pg_fetch_assoc($result, ?int $row_number = null): array|false {}
+function pg_fetch_assoc($result, ?int $row = null): array|false {}
 
 /**
  * @param resource $result
  */
-function pg_fetch_array($result, ?int $row_number = null, int $result_type = PGSQL_BOTH): array|false {}
+function pg_fetch_array($result, ?int $row = null, int $mode = PGSQL_BOTH): array|false {}
```

Comment:
Is there a fundamental difference between `$mode` and `$result_mode` in `mysqli_query()`? If there's not, we could use 1 of the 2 names.

Reply:
Yes, $result_mode in mysqli is the use/store mode. The mode here is what PDO calls the $fetchMode (though just $mode in fetch APIs) and what mysqli used to call the result type ^^